test/cri-containerd/container_test.go

```diff
@@ -476,3 +476,168 @@ func Test_RunContainer_ZeroVPMEM_Multiple_LCOW(t *testing.T) {
 	startContainer(t, client, ctx, containerIDTwo)
 	defer stopContainer(t, client, ctx, containerIDTwo)
 }
+
+func Test_RunContainer_GMSA_WCOW(t *testing.T) {
+	requireFeatures(t, featureWCOWProcess, featureGMSA)
```

Comment:
I would like by convention to leave a blank line after our "requirements" section in each test.